sdk/tests/core/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ include(AddTestCMocka)
 
 # -ld link option is only available for gcc
 if(UNIT_TESTING_MOCKS)
-    set(WRAP_FUNCTIONS "-Wl,--wrap=az_platform_clock_msec -Wl,--wrap=az_http_client_send_request")
```

Comment:
Why did we get rid of `az_http_client_send_request`?

Reply:
No unit test mocks `az_http_client_send_request`. It was only the Client Secret credential that did.